### PR TITLE
Fix demo and css for selection

### PR DESCRIPTION
**Pull request by EWhite613**
### This project uses [semver](semver.org), please check the scope of this pr:

- [ ] #none# - documentation fixes and/or test additions
- [x] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

Selection:
screen shot 2017-06-09 at 1 32 36 pm

Regular table (to ensure no regressions):
screen shot 2017-06-09 at 1 33 02 pm

# CHANGELOG
 * Fix mirage for Demo page
* Fix CSS for Table Selection. Fixes https://github.com/ciena-frost/ember-frost-table/issues/20

### CODE REVIEW

File: tests/dummy/config/environment.js [hunk 1]
```diff
@@ -44,6 +44,10 @@ module.exports = function (environment) {
   if (environment === 'production') {
     ENV.locationType = 'hash'
     ENV.rootURL = '/ember-frost-table'
+    ENV.mirageNamespace = 'https://ciena-frost.github.io'
+    ENV['ember-cli-mirage'] = {
+      enabled: true
+    }
```
Comment (job13er) [inline tests/dummy/config/environment.js]: Thanks for fixing this, I wasn't thinking about the `gh-pages` when I turned on mirage in my other PR

**Comment by job13er** [comment]
Since we don't have visual acceptance going, it would be nice to see a screenshot of the demo rendered after this change in the description (just make sure it's above the `#CHANGELOG` section.

**Comment by EWhite613** [comment]
@job13er updated PR

**Comment by job13er** [comment]
Approved

**Pull request merged**